server/terraform/modules/myhealth/main.tf

```diff
@@ -171,6 +175,10 @@ resource "google_compute_url_map" "urlmap" {
   path_matcher {
     name            = "all"
     default_service = google_compute_backend_service.backend.self_link
+    path_rule {
+      paths   = ["/content/*"]
```

Comment:
So `covid19app.who.int/content` will be the production url? We don't have a website yet so a separate subdomain for static content isn't as much of a concern.

The app currently fetches the content from a Google Storage managed url. I don't have a clear opinion on which is the better approach except that we should probably use only one of them.
https://storage.googleapis.com/who-myhealth-staging-static-content-01

From:
https://github.com/WorldHealthOrganization/app/blob/master/client/lib/api/endpoints.dart#L8-L9

Reply:
Yes, that is what this is configuring here. It would allow access via either path (storage.googleapis or who.int). 
Not sure of best practices for separate subdomains (rpc vs static). My reading of the bug was to route everything through the LB.  And my assumption is that this content is not user created so I can't see any major issues with sharing.
Certainly if there is a desire to front/move everything on a different system (e.g. azure) using a custom domain seems better than storage.googleapis...

Reply:
The system is pretty heavily tied to GCP already.

The benefit of the storage url is that everything is managed for you and probably less likely to fail. Slight downside is that the domain isn't associated with WHO so it could lead to different filtering. I didn't mean for the bug to take a stance on using the LB or not. Perhaps that best so everything goes through the same domain.

Reply:
After this change, both options become (and remain) available. I'm fine either way and can remove this or leave it in.

Reply:
Leave it in for now and let's discuss with the Google Cloud expert they introduced us to